utils/build.mjs

```diff
@@ -40,7 +40,7 @@ export const processActionYml = (
             to: "${{ github.action_path }}/",
         },
     ];
-    Object.keys(packageJsons[name].dependencies ?? {}).forEach((depName) => {
+    Object.keys(packageJsons).forEach((depName) => {
```

Comment:
Is `packageJsons` now a list of dependencies? Or is this just saying that each package is a dependency to all the other packages?

Reply:
`packgeJsons` is the full map of `action name` to `contents of the package.json`

Reply:
So, this code is doing what? I guess I'm trying to understand how the keys of `packageJsons` maps to `depName`. What is this doing exactly?